### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by faweizhao26**

### CODE REVIEW

**Comment by faweizhao26** [comment]
@LinuxSuRen

File: README.md [hunk 1]
```diff
@@ -134,3 +136,4 @@ Teahour.fm 专注程序员感兴趣的话题，包括 Web 设计和开发，移
 [37]: https://github.com/opensource-f2f/devops-talk, 'DevOps Talk'
 [38]: https://xiaoyuzhoufm.com/podcast/6301f1f98063c4980da9806a, '科技双眼啤'
 [39]: https://xiaoyuzhoufm.com/podcast/61d4278c2654166e94d07d1f, 'NebulaGraph 星球'
+[40]: https://www.xiaoyuzhoufm.com/podcast/6375d6e8b0725ebdf1e702e6, ‘KubeSphere Talk'
```
Comment (LinuxSuRen) [inline README.md:139]: It looks like `‘KubeSphere Talk'` is incorrect. It should be `'KubeSphere Talk'`

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (faweizhao26) [reply README.md]: > It looks like `‘KubeSphere Talk'` is incorrect. It should be `'KubeSphere Talk'`

Fixed.

**Comment by LinuxSuRen** [comment]
> Thank you for your job. You may need copy https://github.com/tech-podcasts/it-technology-podcast/blob/main/website/docs/podcast/_%E6%A8%A1%E6%9D%BF.mdx to `/website /docs/ podcast /`, and rename.

We could figure out a way to avoid maintaining the README file manually.

**Pull request closed without merging**